apps/lockfile-explorer-web/src/containers/LockfileEntryDetailsView/index.tsx

```diff
@@ -231,6 +258,24 @@ export const LockfileEntryDetailsView = (): JSX.Element | ReactNull => {
     );
   };
 
+  const getDependencyInfo = (
+    rawEntryId: string,
+    entryPackageName: string
+  ): { type: DependencyKey; version: string } | undefined => {
+    const packageJson = directRefsPackageJSON.get(rawEntryId);
+    if (!packageJson) return undefined;
+
+    const dependencyTypes = [DependencyKey.DEP, DependencyKey.DEVDEP, DependencyKey.PEERDEP];
```

Comment:
A package.json file may specify the same dependency as BOTH a regular dependency AND a devDependency, or BOTH a peer dependency AND a devDependency.  This is useful when the published NPM package has a broad range, but we want a specific range for testing. For example:

**package.json**
```
  "dependencies": { "react": "^18.0.0" },
  "devDependencies": { "react": "18.2.3" }
```


**package.json**
```
  "peerDependencies": { "react": "^17.0.0 || ^18.0.0" },
  "devDependencies": { "react": "17.3.4" }
```

Reply:
This situation seems reasonable according to the current parsing order.？follow `dependencies -> devDependencies -> peerDependencies` order